cla_public/apps/checker/validators.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, message=None, freq_message=None, amount_message=None):
         self.freq_message = freq_message
         self.amount_message = amount_message
 
-    def append_interval_text(self, field):
+    def append_interval_text(self, field, interval):
```

Comment:
Oh yes, I spotted this when we were writing the signature but planned to lead you towards finding it yourself and then completely forgot 🤦 😅